pkg/rewards/claim.go

```diff
@@ -216,7 +343,7 @@ func Claim(cCtx *cli.Context, p utils.Prompter) error {
 			noSendTxOpts.GasLimit = 150_000
 		}
 
-		unsignedTx, err := contractBindings.RewardsCoordinator.ProcessClaim(noSendTxOpts, elClaim, config.RecipientAddress)
+		unsignedTx, err := contractBindings.RewardsCoordinator.ProcessClaim(noSendTxOpts, *elClaim, config.RecipientAddress)
```

Comment:
Do we need `ProcessClaim` anymore? I think all this can be really simplified if we use `ProcessClaims` for both batch and normal claim. I am pretty sure `ProcessClaims` works for single earner. I think contract devs have left it for backward compatibility if I am not wrong.

Reply:
I agree there would be benefits to being able to use just the batch claim method in a refactor. I will check with @0xrajath if we should do all claims through the batch claim even single claim (maybe gas reasons?). I kept it separate to reduce the likelihood of introducing bugs into the existing single claim flow and mirror upstream contract architecture.

Reply:
I see - yea I don't know about gas implications tbh. something to find out.

Reply:
@shrimalmadhur and I discussed and we will keep separate to make testing easier when code is in testnet but not yet mainnet. The single `processClaim` is available in mainnet since it is an existing method.